addons/escoria-core/game/core-scripts/esc_game.gd

```diff
@@ -14,6 +14,14 @@ enum EDITOR_GAME_DEBUG_DISPLAY {
 }
 
 
+# The main menu node
+export(NodePath) var main_menu
+
+
```

Comment:
Only one empty line here and in the other export (two lines denote a section break and all these are export variables)

Reply:
Fixed